### DISCUSSION TITLE
How do i recover an account whose email i have no longer access?

### DISCUSSION THREAD

**Discussion by noa-stavrou** [community/community#67018] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do i recover an account whose email i have no longer access to? 

Every github support page I have visited to help me with this, ends up at the same place, where I have to enter a security code sent to that email!!!

Our company changed name, so my email from user@company-a.com was changed to user@company-b.com and the company-a.com email server was deleted. 

Any help would be greatly appreciated.

**Comment by hafiz-muhammad** [comment]
If you have recovery codes you can use those to regain access to your account. It's always a good idea to add a backup email address to your account.

Here are several GitHub documentations about recovering your account:
- [GitHub Account Recovery Policy](https://docs.github.com/en/site-policy/other-site-policies/github-account-recovery-policy)
- [Recovering your account if you lose your 2FA credentials](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials)

**Comment by noa-stavrou** [comment]
Thanks for the quick response.

I do not have any recovery codes, I don't event think 2FA was used in the account.

I still have username and password, the problem is when I log in, it sends me to a page to enter a security code sent to that email account I no longel have access to.

**Reply by hafiz-muhammad** [reply, marked as answer]
You're welcome. Since you don't think you've used 2FA on the account, you can try and contact GitHub explaining the issue to them.

Use the link below to submit a ticket to GitHub support for account related issues:
[https://support.github.com/contact/account](https://support.github.com/contact/account)

**Reply by judithmiranda** [reply]
i am having same issue, how did you manage to log in?

**Reply by mjr0108** [reply]
I also have the issue and you can't use the support without the access code either.  Seems like there is no help at all for what appears to be a fairly common issue

**Comment by huywashere** [comment]
hi

**Comment by nidalwatfa** [comment]
URGENT: Account at Risk - Token Leaked - 2FA Lockout in Restricted Region (Syria)
​Account Username: nidalwatfa
Status: High Risk (Personal Access Token Leaked)
​Description of the Issue:
I am writing this to request immediate intervention from the GitHub Security/Support team. My account is currently at severe risk as my Personal Access Token has been leaked, and I am unable to secure the account or revoke access due to a mandatory 2FA lockout.
​Barriers Encountered:

​Mandatory 2FA Enforcement: I am being forced to enable 2FA to proceed, but I am located in Syria, which is not included in the supported countries list for SMS authentication.

​Recovery Code Failure: Despite having valid recovery codes (as shown in my records), every attempt to use the /sessions/two-factor/recovery path redirects me to help documentation (docs.github.com) instead of providing an input field.

​App/Web Loop: The interface remains stuck in a loop asking to "Enable 2FA" but provides no functional way to bypass this using existing recovery assets or email verification in my region.

​Requested Action:
Due to the security breach (leaked token), I urgently request that GitHub Support:

​Manually disable the 2FA requirement for my account temporarily so I can log in and secure my data.

​Alternatively, provide a direct secure link to revoke all active tokens and sessions.

​I have all the necessary documentation and recovery codes to prove ownership, but the automated system is failing to provide the interface to use them.

**Comment by narutosenkapk-Offical** [comment]
This is a helpful question, and I'm sure others who have lost access to an old email will find it useful. The explanation of the issue with company email changes makes the situation easy to understand, and it's great that the discussion includes possible recovery options. I was browsing similar tech topics and also came across [narutosenkiapk.app](https://narutosenkiapk.app/), which has interesting content for mobile gaming fans. Hopefully anyone facing this problem can recover their account successfully.

**Comment by AdityaNMGHUB** [comment]
Hello GitHub Support,

I am requesting assistance in recovering access to my GitHub account. Unfortunately, I no longer have access to the email address associated with the account, so I am unable to receive verification codes, password reset emails, or complete the account recovery process.

This GitHub account is important to me, as it contains my repositories, contributions, and development history. I understand the importance of account security and am fully willing to provide any information necessary to verify my identity and ownership of the account. This may include my GitHub username, the previously associated email address, repository names, approximate account creation date, commit history, or any other details you require.

Once my identity has been verified, I would greatly appreciate your assistance in restoring access to my account or updating the associated email address to one that I currently control.

Thank you for your time and support. I appreciate your assistance and look forward to your guidance on the next steps.

Kind regards,

Aditya Narayan Mishra

GitHub Username: AdityaNMGHUB
